lib/src/serializable.dart

```diff
@@ -138,6 +143,7 @@ class SerializableBus {
   void _deserializeAndCall(
       SerializableModule module, String method, List data) {
     InstanceMirror apiMirror = reflect(module.api);
+    List reflectedData = new List(data.length);
```

Comment:
When compiling with DDC, the original list `data` gets a runtime type of `List<Map<String, String>>`, which throws a runtime error when trying to assign a `JsonSerializable` to `data[i]`. I tried changing the type of `List<dynamic>` but that didn't work.

Adding the new list removes that type check and allows the list to be a `List<JsonSerializable>`